.travis.yml

```diff
@@ -13,3 +13,25 @@ script:
   - yarn pretty-check
   - yarn build
   - yarn test
+before_deploy:
+  - export PACKAGE_VERSION=$(jq -r '.version' package.json)
+  - test "v$PACKAGE_VERSION" = "$TRAVIS_TAG"
+  - mkdir -p deploy
+  - yarn pack --filename deploy/dex-liquidity-provision-$PACKAGE_VERSION.tgz
+deploy:
+  provider: s3
+  bucket: $BUCKET_NAME
+  access_key_id: $AWS_ACCESS_KEY_ID
+  secret_access_key: $AWS_SECRET_ACCESS_KEY
+  local_dir: deploy
+  skip_cleanup: true
```

Comment:
If possible I would prefer not to skip cleanup.

Reply:
@nlordell Why is the clean_up skipped in dex-contracts?

Reply:
@nlordell Do you know why the clean up is skipped?

Reply:
https://docs.travis-ci.com/user/deployment-v2#cleaning-up-the-git-working-directory I assume we need the artefacts that were generated in the prepack step and are not part of the repo. If we cleaned up before deployment all those would be reset.